<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kommivoyazher</class>
 <widget class="QMainWindow" name="Kommivoyazher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача коммивояжёра</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.523, x2:1, y2:0.511, stop:0.512438 rgba(253, 255, 201, 255), stop:0.676617 rgba(184, 251, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i6">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>17</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j3">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j2">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j1">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j4">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i2j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i1j4">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>315</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i4j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>17</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i5j5">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>495</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j6">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>225</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i3j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>405</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i0j3">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;
background-color:rgb(254, 255, 217)</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>716</x>
      <y>115</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 0)
}

QPushButton:pressed
{
background-color:rgb(235, 231, 101)
}
</string>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_show">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>15</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
background-color:rgb(255, 255, 174);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_number">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>105</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 174);
font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sum">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>165</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 174);
font: 14pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>115</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Введите номер узла</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Длина пути</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>281</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Задача коммивояжёра</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>❕Если между узлами нет связей, то ставить значение &quot;-1&quot;. </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
